_random.py

```diff
@@ -0,0 +1,53 @@
+import random
+
+def int(start, end, count=1):
+    """ 
+    A function to return 'n' random integers between a given range 'Start' to 'End'
+        parameters: 
+        start - start of the range
+        end - end of the range (inclusive)
+        count(optional) - number of random integers to return, default value is 1
+        Note: count <= end-start
+    """
+
+    if end-start < count:
+        raise ValueError("Count of the integers should not be greater than the input range !")
+    if count>1:
```

Comment:
Put spaces around operators (<, >, -, +, *, %, etc...). Not only on this line, but there are also a few other places.